lambdas/models/config.py

```diff
@@ -7,3 +7,7 @@ def to_camel(string: str) -> str:
 
 
 conf = ConfigDict(alias_generator=to_camel)
+
+
+def to_capwords(snake_case_string: str) -> str:
+    return inflection.camelize(snake_case_string, uppercase_first_letter=True)
```

Comment:
Quick question on this function, seems it's doing the same as the `to_camel` function above. What's the output we are looking for here that makes it different?

Reply:
`to_camel` has uppercase_first_letter=False and this one got uppercase_first_letter=True . 
I tried `to_camel` but it doesn't fit the current schema we are using so made this one.

Reply:
Ah I see you're right, in that case maybe change the name of the function to `to_capitalized_camel`? `capwords` suggests all letters are capital that's all but I'll leave that up to you

Reply:
Nice one. Agree that `capwords` is kind of confusing. Renamed to `to_capitalized_camel` :D